docs/release-notes/cloud/cloud-2023-10.md

```diff
@@ -0,0 +1,81 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud October 2023.
+included_jira_versions: ['2023.10']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.10.32
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.10.80
+---
+
+# Cloud October 2023
+
+These release notes are for the Codacy Cloud updates during October 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   You can now get two additional coverage status reports on GitHub using the new faster Coverage engine, showing whether your diff coverage and coverage variation are up to standards or not as configured on the quality gate rules for your repository. See [how to activate this setting](../../repositories-configure/integrations/github-integration.md#new-coverage-engine) for a repository. (ALA-593)
+
+    ![New Coverage status report](../images/ala-593.png)
+
+-   Codacy will drop support for branches named HEAD or matching the pattern refs/heads/*. These names are reserved for git and shouldn't be used as branch names. (IO-830)
```

Comment:
@claudiacarpinteiro Can you review this one? 🙏

Reply:
```suggestion
-   Codacy no longer supports and now skips the analysis of branches named `HEAD` or matching the pattern `refs/heads/*`, as these are Git reserved terms. (IO-830)
```

Reply:
I'm, not sure if we want to announce this as a product enhancement...

@MiguelMarcelino WDYT?
Does this change fit a product enhancement, or should we announce it as a bug fix? 🤔

Reply:
Not entirely sure. For me, this looks like a bug. However, I would say that calling it a bug vs. enhancement depends on what we consider to be the source of truth. If the provider is the source of truth, then it is an enhancement. Despite our supported providers allowing us to create these branches, some providers don't behave that well when this is done. If we want to use git as the source of truth, then this is a bug.
But I would maybe ask someone from @codacy/io to give their opinion about this as well.

Reply:
I would tend to agree with @MiguelMarcelino's initial assessment, this is a bug imo